src/main/java/org/fsg1/fmms/backend/endpoints/CurriculumEndpoint.java

```diff
@@ -40,6 +38,7 @@ public CurriculumEndpoint(final CurriculumService curriculumService) {
     @Produces(MediaType.APPLICATION_JSON)
     public Response getCurriculumSemesters(@PathParam("curriculum_id") final int curriculumId) {
         try {
+            final CurriculumService service = (CurriculumService) getService();
             final JsonNode result = service.get(service.getQueryCurriculumSemestersString(), curriculumId);
             final String jsonString = result.toString();
             return Response.status(Response.Status.OK).entity(jsonString).build();
```

Comment:
Please append information about the exception to the request. At least the exception message.

Reply:
That's a thing for later maybe. If Sjoerd doesn't need exception information I won't put it in :+1: